### ISSUE TITLE
Goal of this repository

### ISSUE THREAD

**Issue by ktk** [periodo/edtf-literals#1]
Let's get some ideas on what we could do here. From my POV:

- Have a place where groups interested in EDTF-related time formats & RDF can discuss & share know-how.
- Provide code and/or references to code that helps convert various time formats to EDTF or vice versa
- Have a place to talk about OWL time in the context of EDTF, provide examples of mapping.
- Talk about integrating/using EDTF & OWL Time in SPARQL

We might use the new GitHub discussion feature for that, I think issues might be a bit too formal in the beginning at least.

**Comment by rybesh** [comment]
> Have a place where groups interested in EDTF-related time formats & RDF can discuss & share know-how.

Absolutely. I’ve never used GitHub Discussions before, but it seems suited for that.

> Provide code and/or references to code that helps convert various time formats to EDTF or vice versa

Probably references to code, since putting a bunch of loosely related code in one repo could get unwieldy.

> Have a place to talk about OWL time in the context of EDTF, provide examples of mapping.

Yes. I would go further and say not just examples of mapping, but trying to build some consensus on standards ways of translating EDTF expressions to “expanded” representations modeled using OWL-Time (and possibly CIDOC CRM as well, though I would want to start with OWL-Time).

> Talk about integrating/using EDTF & OWL Time in SPARQL.

Yes. This might also involve some discussion of how to implement rules for translating back and forth between EDTF literals and “expanded” triple sets, so that data using EDTF literals can be queried over using non-EDTF-aware SPARQL engines (and then “compressed” back to EDTF if necessary for export as e.g. CSV). Something like [N3 entailment rules](https://n3.restdesc.org) seem suited for this, but I’m not an expert in this area.

**Comment by ktk** [comment]
Ok should we move this issue to discussion as well? See https://github.com/github/feedback/discussions/2952
